### PR TITLE
Misc docs

### PR DESCRIPTION
**Pull request by NicolasHug**

### CODE REVIEW

File: torchvision/transforms/v2/_color.py [hunk 1]
```diff
@@ -15,17 +15,11 @@ class Grayscale(Transform):
 
     .. betastatus:: Grayscale transform
 
-    If the image is torch Tensor, it is expected
-    to have [..., 3, H, W] shape, where ... means an arbitrary number of leading dimensions
+    If the input is a Tensor, it is expected
```
Comment (NicolasHug) [inline torchvision/transforms/v2/_color.py:18]: changes in this file are drive-bys

Comment (vfdev-5) [inline torchvision/transforms/v2/_color.py:18]: yeah, maybe also 
```suggestion
    If the input is a :class:`torch.Tensor`, it is expected
```

File: torchvision/transforms/v2/_misc.py [hunk 2]
```diff
@@ -247,6 +254,34 @@ def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
 
 
 class SanitizeBoundingBoxes(Transform):
+    """[BETA] Remove degenerate/invalid bounding boxes and their corresponding labels.
```
Comment (pmeier) [inline torchvision/transforms/v2/_misc.py:257]:
```suggestion
    """[BETA] Remove degenerate/invalid bounding boxes and their corresponding labels and optionally masks.
```

This also needs to go into the text below.

**Review by pmeier — APPROVED** [review]
Masks are missing from `SanitizeBoundingBoxes` but otherwise LGTM!
